src/main/setup.js

```diff
@@ -33,10 +33,21 @@ ipcMain.handle("save-logs", async (event, path: { canceled: boolean, filePath: s
 
 ipcMain.handle(
   "export-operations",
-  async (event, path: { canceled: boolean, filePath: string }, csv: string) => {
-    Promise.resolve().then(
-      () => !path.canceled && path.filePath && csv && fsWriteFile(path.filePath, csv),
-    );
+  async (event, path: { canceled: boolean, filePath: string }, csv: string): Promise<boolean> => {
+    try {
+      if (!path.canceled && path.filePath && csv) {
+        await fsWriteFile(path.filePath, csv);
+        return true;
+      }
+
+      if (path.canceled) {
+        return false;
+      }
+
+      return false;
```

Comment:
```suggestion
```

This part has no use ("if canceled return false, otherwise return false, then return false" 🤔)